### DISCUSSION TITLE
Can analyze shotgun use input file name expansion?

### DISCUSSION THREAD

**Discussion by DawnChou** [milaboratory/mixcr#1071] [Q&A] [answered]
Hi,
I'm trying to use MiXCR for antigen receptor calling of TCRbeta, and I have two Read 2 files from different lanes.  _GSM5416366_patient_16_S1_R2_L001.fastq.gz_ and _GSM5416366_patient_16_S1_R2_L002.fastq.gz_
I found [https://mixcr.com/mixcr/reference/ref-input-file-name-expansion/](url) says {{n}} can match any number, so I input
`mixcr analyze shotgun /public/yipingzou2/spatialTCR/GSM5416366_patient_16_S1_R2_L{{n}}.fastq.gz mixcr_output_3/clones --species hsa --starting-material rna --align "-OsaveOriginalReads=true" --assemble "--write-alignments"`

However, it returns
 `ERROR: input file "/public/yipingzou2/spatialTCR/GSM5416366_patient_16_S1_R2_L{{n}}.fastq.gz" does not exist.`

I checked the prefix again and again, but I did not find anything wrong. I'm new to mixcr and am unsure if I can use the input file name expansion here. Thanks for your help.

**Comment by mizraelson** [comment]
Hi,
What MiXCR version do you use?

**Comment by DawnChou** [comment]
> Hi, What MiXCR version do you use?

I am using mixcr 3.0.13

**Comment by mizraelson** [comment]
The Lane merging feature is only available starting v4. I recommend updating to the latest version.

**Comment by DawnChou** [comment]
> The Lane merging feature is only available starting v4. I recommend updating to the latest version.

OK. Thank you!

**Comment by mizraelson** [comment, marked as answer]
Also please note that some commands have changed and `mixcr analyze shotgun` has been deprecated.

For RNA-seq data the correct command will be:

```
mixcr analyze rnaseq-full-length \
    --species hsa 
    /public/yipingzou2/spatialTCR/GSM5416366_patient_16_S1_R2_L{{n}}.fastq.gz \
    mixcr_output_3/clones
```

You can read more here:
https://mixcr.com/mixcr/reference/overview-built-in-presets/#rna-seq-data
